### ISSUE TITLE
Need a logo for IGN

### ISSUE THREAD

**Issue by Wabri** [Schrodinger-Hat/ImageGoNord#100]

**Comment by Nareshchoudhary02** [comment]
![default](https://user-images.githubusercontent.com/107427909/215344904-d0072a44-5278-42ef-ac0e-d471aadccbe4.png)
<img width="914" alt="Logo" src="https://user-images.githubusercontent.com/107427909/215344908-3f873283-1c06-4124-8337-fd0338c73dab.png">

@Wabri  `Please review and let me know your thoughts`

**Comment by Wabri** [comment]
Hi @Nareshchoudhary02 those are not on the verve of the project.

First of all the logo need to use the https://www.nordtheme.com/ palette and the image is not something that I see and associate to something that convert rgb image to nord palette image.

Did you try our service? https://ign.schrodinger-hat.it/

**Comment by Nareshchoudhary02** [comment]
> 
yes I have tried `Image Go Nord` service and its great tool to make clean Nord image. Thanks for your reviews, I will try to make actual logo rather than an image 🙂.

**Comment by Wabri** [comment]
Thank you for your contribution, I really appreciate it 'cause I'm incapable of doing graphics 😬

**Comment by TheJoin95** [comment]
@Wabri what's the progress on this? Are we still in the need of having a logo?

**Comment by Wabri** [comment]
I dont know, I would like to, but I've not the competence on doing it. I've created with `help wanted` for a reason 😁  Close if you wish!
